tests/baselines/reference/moduleAugmentationDeclarationEmit1.js

```diff
@@ -0,0 +1,68 @@
+//// [tests/cases/compiler/moduleAugmentationDeclarationEmit1.ts] ////
+
+//// [map.ts]
+
+import { Observable } from "./observable"
+
+(<any>Observable.prototype).map = function() { }
```

Comment:
this is just an example that in module `map` somebody can patch the prototype of `Observable` and add new functionality to it and *also* describe what was done to our compiler via module augmentation. What was added to the prototype and how it was done is not really important here.